gym_wrapper/base_gym_env.py

```diff
@@ -0,0 +1,241 @@
+import os
+from typing import List, Optional
+
+import gym
+import numpy as np
+import pygame
+import vizdoom.vizdoom as vzd
+from gym import spaces
+from vizdoom import scenarios_path
+
+
+
+CONFIGS = [
```

Comment:
With "dynamic" action space shaping, I feel these config names could be directly in the env class definitions, instead of just indices (makes it so much readable).